libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -981,17 +981,26 @@ AP_GPS_UBLOX::_parse_gps(void)
                         gnssCount++;
                     }
                 }
-
                 for(int i = 0; i < _buffer.gnss.numConfigBlocks; i++) {
-                    // Reserve an equal portion of channels for all enabled systems
+                    // Reserve an equal portion of channels for all enabled systems that supports it
                     if(gps._gnss_mode[state.instance] & (1 << _buffer.gnss.configBlock[i].gnssId)) {
-                        if(GNSS_SBAS !=_buffer.gnss.configBlock[i].gnssId) {
+
+                        if(GNSS_SBAS !=_buffer.gnss.configBlock[i].gnssId && GNSS_GALILEO !=_buffer.gnss.configBlock[i].gnssId) {
```

Comment:
You only want to special case galileo if the hardware version is less then or equal to a M8, I need to validate it still, but I'd be surprised if the restriction still applies to the F9P and similar units.